### PR TITLE
Auto disable logs after 14 days

### PR DESCRIPTION
**Pull request by Nethius**

### CODE REVIEW

**Review by Noxius87 — APPROVED** [review]

File: client/ui/controllers/settingsController.cpp [hunk 1]
```diff
@@ -1,6 +1,7 @@
 #include "settingsController.h"
 
 #include <QStandardPaths>
+#include <QtConcurrent>
```
Comment (pokamest) [inline client/ui/controllers/settingsController.cpp]: Please remove unused include

File: client/ui/controllers/settingsController.cpp [hunk 2]
```diff
@@ -27,6 +28,7 @@ SettingsController::SettingsController(const QSharedPointer<ServersModel> &serve
       m_settings(settings)
 {
     m_appVersion = QString("%1 (%2, %3)").arg(QString(APP_VERSION), __DATE__, GIT_COMMIT_HASH);
+    startLoggingWather();
```
Comment (pokamest) [inline client/ui/controllers/settingsController.cpp]: Please rename function to something like checkIfNeedDisableLogs

File: client/ui/controllers/settingsController.h [hunk 3]
```diff
@@ -76,6 +76,8 @@ public slots:
 
     void amneziaDnsToggled(bool enable);
 
+    void loggingDisableByWathcer();
```
Comment (pokamest) [inline client/ui/controllers/settingsController.h]: Please fix name to correct spelling

**Review by pokamest — CHANGES_REQUESTED** [review]

File: client/ui/controllers/settingsController.cpp (hunk 1 above)
Comment (agalehaga) [reply client/ui/controllers/settingsController.cpp]: Fixed)

File: client/ui/controllers/settingsController.cpp (hunk 2 above)
Comment (agalehaga) [reply client/ui/controllers/settingsController.cpp]: Fixed)

File: client/ui/controllers/settingsController.h (hunk 3 above)
Comment (agalehaga) [reply client/ui/controllers/settingsController.h]: Fixed)

File: client/settings.cpp [hunk 4]
```diff
@@ -224,11 +224,22 @@ void Settings::setSaveLogs(bool enabled)
         if (!Logger::init()) {
             qWarning() << "Initialization of debug subsystem failed";
         }
+        setLogEnableDate(QDateTime::currentDateTime());
```
Comment (pokamest) [inline client/settings.cpp:230]: why it's called in #ifndef Q_OS_ANDROID?
Looks like need to fix

**Review by pokamest — CHANGES_REQUESTED** [review]

**Review by pokamest — APPROVED** [review]

**Pull request merged**